src/environment_setup/tool_providers.ml

```diff
@@ -319,22 +331,31 @@ let kallisto =
 let samtools =
   let url = "https://github.com/samtools/samtools/releases/download/1.3/\
              samtools-1.3.tar.bz2" in
-  let toplevel_tools = ["samtools"] in
   let htslib = ["bgzip"; "tabix" ] in
-  let tools = toplevel_tools @ htslib in
   let install_program ~path =
     let open KEDSL.Program in
     sh "make"
-    && shf "cp %s %s" (String.concat toplevel_tools ~sep:" ")  path
+    && shf "cp samtools %s"  path
```

Comment:
This I could definitely change back; I just didn't like the concat of 1 going on. Though now I repeat "samtools"…